packages/utils/src/istanbul.ts

```diff
@@ -0,0 +1,62 @@
+import BigNumber from 'bignumber.js'
+import { toChecksumAddress } from 'ethereumjs-util'
+import * as rlp from 'rlp'
+import { Address } from './address'
+
+// See https://github.com/celo-org/celo-blockchain/blob/master/core/types/istanbul.go
+export const ISTANBUL_EXTRA_VANITY_BYTES = 32
```

Comment:
To avoid having the utils exports overcrowded, the general strategy has been to encapsulate each file's utils in a wrapper object and export that instead. See https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/src/phoneNumbers.ts#L234
and
https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/src/index.ts#L8

Admittedly we haven't always been great about this but we should be better as this package grows

Reply:
Thanks for  letting me know! I changed it